src/test/kotlin/com/terraformation/backend/email/EmailNotificationServiceTest.kt

```diff
@@ -820,6 +829,47 @@ internal class EmailNotificationServiceTest {
     assertRecipientsEqual(setOf("support@terraformation.com"))
   }
 
+  @Test
+  fun `deliverableReadyForReview with Terraformation contact`() {
```

Comment:
Worth testing the "TF contact is also an accelerator admin and shouldn't be double-notified" case?